packages/panels/src/components/sidebar-menu/SidebarMenu.tsx

```diff
@@ -24,15 +25,21 @@ export const SidebarMenu: React.FC<SidebarMenuProps> = ({
   className,
   children,
   onCloseClick,
+  hideCloseButton,
   ...boxProps
 }) => {
   return (
     <Box className={cx(styles.sidebarMenu, className)} {...boxProps}>
-      <Box alignItems={"flex-start"} justifyContent={"center"}>
-        <SidebarMenuCloseButton onClick={onCloseClick} variant={"dark"} />
-      </Box>
-      <SeparatorLine color={cssColor("--lhds-color-blue-700")} />
-      <Space />
+      if (!hideCloseButton){" "}
+      {
+        <>
+          <Box alignItems={"flex-start"} justifyContent={"center"}>
+            <SidebarMenuCloseButton onClick={onCloseClick} variant={"dark"} />
+          </Box>
+          <SeparatorLine color={cssColor("--lhds-color-blue-700")} />
+          <Space />
+        </>
+      }
```

Comment:
You need to wrap your js code in `{}`, otherwise it will be treated as regular html, e.g. text.

Also, you can shorthand `if` with `&&`. If `!hideCloseButton` is true, it will render the stuff after `&&`.

```suggestion
      {!hideCloseButton && (
        <>
          <Box alignItems={"flex-start"} justifyContent={"center"}>
            <SidebarMenuCloseButton onClick={onCloseClick} variant={"dark"} />
          </Box>
          <SeparatorLine color={cssColor("--lhds-color-blue-700")} />
          <Space />
        </>
      )}
```